src/systems/systems/src/hydraulic/brake_circuit.rs

```diff
@@ -448,7 +444,8 @@ impl AutobrakeDecelerationGovernor {
     fn disengage(&mut self) {
         self.is_engaged = false;
         self.time_engaged = Duration::from_secs(0);
-        self.target = Acceleration::new::<meter_per_second_squared>(10.);
+        self.pid_controller
+            .reset(Self::RESET_ACCEL_VALUE_M_S_SQUARED);
```

Comment:
**question**: Would a particular instance of a `PidController` always be reset to the same value, or can the value be different?
**suggestion**: If it's always reset to the same value I'd suggest moving the reset value into the constructor function and then simply writing `self.pid_controller.reset()` here and at the other usage.

Reply:
It might be useful to chose the reset point indeed for some cases. But your remark makes me realize code is wrong :D  The pid output is not necessarily same type as input, and in that case it's not, so it should be reset to 0 not to an acceleration value.

Reply:
Removed the parameter as this can be handled through a setpoint change when updating the controller.